recipes/pandas-gbq/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: pandas-gbq
+  version: {{ version }}
+
+source:
+  fn: pandas-gbq-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/source/p/pandas-gbq/pandas-gbq-{{ version }}.tar.gz
+  md5: e59cff308781cbdbfcaec7fa53e51f7d
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - pandas
+
+test:
+  imports:
+    - pandas_gbq
+
+about:
+  home: https://github.com/pydata/pandas-gbq
+  license: BSD
```

Comment:
Please ask upstream to add `LICENSE.md` to the `MANIFEST.in` so that it will packaged in `sdist`s in the future. For now, please add `# license_file: LICENSE.md`. Once the license file is added this line can then be uncommented and the license added to the conda package.